plugins/tiddlywiki/codemirror/engine.js

```diff
@@ -111,7 +111,11 @@ function CodeMirrorEngine(options) {
 
 	console.log("config: ", config)
 
-	config.mode = options.type;
+  if (options.type == "text/vnd.tiddlywiki" || options.type == "text/x-tiddlywiki") {
+    config.mode = "text/html";
+  } else {
+    config.mode = options.type;
+  }
 	config.value = options.value;
```

Comment:
I'd prefer to change meta.js with the right settings. So imo no hack would be needed. .. We modified meta.js anyway. So it should probably get a new name, since it isn't original anymore.

Reply:
that's good. I don't know the right settings, I tried before

tiddlymeta.js a good name?
I'll grep what's using it and rename when I'm back - then I'll also revert the hack